app/src/main/java/org/mozilla/tv/firefox/pocket/PocketVideoFragment.kt

```diff
@@ -83,23 +82,16 @@ class PocketVideoFragment : Fragment() {
         compositeDisposable.clear()
     }
 
-    // When menu is pressed from the Pocket feed, go back to overlay
```

Comment:
Did you verify that the behavior in this comment is still present after this change?

Reply:
Yep! This code was actually listed as unused by the compiler so I just removed it.